Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+
+ARG ACLI_CHECKSUM="ec6368888922d4d036917b033261920423352c4c40d702c00a316a6553263e7c"
+ARG ACLI_VERSION="1.22.0"
```

Comment:
Does this need to be updated for every release?

Reply:
An alternative to that is to always pull the latest Acquia CLI release, but then there's no easy checksum verification.

Reply:
No new manual release steps.

Reply:
Let's keep it simple without the checksum then.